### ISSUE TITLE
Poor contrast on headlines in Fronts

### ISSUE THREAD

**Issue by nicl** [guardian/frontend#6408]
## Steps to reproduce

- go to a front
- certain headlines have poor contrast and fail accessibility standards

## Environment

Device: Laptop
Browser: Chrome
OS: Mac OS
URL: http://theguardian.com/uk

## Details

![screen shot 2014-10-10 at 10 14 29](https://cloud.githubusercontent.com/assets/858402/4590376/e60bca4a-505e-11e4-9104-b6cb2dd73a04.png)

It looks okay-ish here, as the image is bigger for some reason, but even on my laptop which has great contrast, it can still be difficult to read (specifically, the light blue font). The light blue seems to fail this accessibility check:

http://webaim.org/resources/contrastchecker/ (with foreground: #4bc6df, background: #f1f1f1)

**Comment by nicl** [comment]
Closing as an unfixed given lack of activity.
